packages/exception/src/http/http.exception.ts

```diff
@@ -2,11 +2,23 @@ import { ErrorMessage } from '@exception/error-message.interface';
 import { ErrorPayload } from '@exception/error-payload.interface';
 import { InternalException } from '@exception/internal.exception';
 
+export interface SerializedHTTPException {
+  message: string;
+  path: string;
+  statusCode: number;
+  statusText: string;
+  timestamp: string;
+}
+
 export class HTTPException extends InternalException {
   public static instanceOf(err: any): err is HTTPException {
     return err instanceof HTTPException;
   }
 
+  public static serializedInstanceOf(err: any): err is SerializedHTTPException {
+    return typeof err === 'object' && err.statusText && err.message && err.statusCode && err.path && err.timestamp;
```

Comment:
this required all the fields not only being set, but also having a truthy value. should we instead do `'prop' in object` ?

Reply:
maybe rename it to `isSerializedInstance` ?

Reply:
Added a few more clauses in additional to the feedback so the check is more robust.